### PR TITLE
List: Sub Text Max Text (Wearable): add example and style

### PR DESCRIPTION
**Pull request by grzegorz-czajkowski**
[Issue] https://github.com/Samsung/TAU/issues/1052
[Problem] Missing examples and styls
[Solution] Add exmaples and styles

Signed-off-by: Grzegorz Czajkowski <g.czajkowski@samsung.com>

### CODE REVIEW

File: examples/wearable/UIComponents/contents/presentation-views/list/main/sub-text/1-line-text-max-fadeout-left.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+		<header class="ui-header">
+			<h2 class="ui-title">
+				Title
+			</h2>
+		</header>
+		<div class="ui-content">
+			<ul class="ui-listview">
+				<li class="li-has-multiline li-fadeout-left">
+					<a>
+						Texttexttextexttextte
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/presentation-views/list/main/sub-text/1-line-text-max-fadeout-left.html:30]: Please change to another text.

File: src/css/profile/wearable/changeable/theme-circle/listview.less [hunk 2]
```diff
@@ -95,6 +95,31 @@
...
 		}
 
+		&.li-fadeout {
+			&-left, &-right {
+				&::after {
+					content: "";
+					position: absolute;
+					width: 34 * @px_base;
+					height: 100%;
+				    background: linear-gradient(90deg, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.52), rgba(0, 0, 0, 1));
```
Comment (DohyungLim) [inline src/css/profile/wearable/changeable/theme-circle/listview.less]: check indent.

File: src/css/profile/wearable/changeable/theme-circle/listview.less [hunk 3]
```diff
@@ -95,6 +95,31 @@
...
+					transform: rotate(-180deg);
+					background-image: linear-gradient(90deg, rgba(0, 0, 0, 0.00) 40%, rgba(0, 0, 0, 0.52) 79%, #000000 100%);
+				}
+			}
+			&-left {
+				&::after {
+					left: 15 * @px_base;
```
Comment (DohyungLim) [inline src/css/profile/wearable/changeable/theme-circle/listview.less]: no need to transform?

File: src/css/profile/wearable/changeable/theme-circle/listview.less [hunk 4]
```diff
@@ -95,6 +95,31 @@
...
+		&.li-fadeout {
+			&-left, &-right {
+				&::after {
+					content: "";
+					position: absolute;
+					width: 34 * @px_base;
+					height: 100%;
+				    background: linear-gradient(90deg, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.52), rgba(0, 0, 0, 1));
+					transform: rotate(-180deg);
+					background-image: linear-gradient(90deg, rgba(0, 0, 0, 0.00) 40%, rgba(0, 0, 0, 0.52) 79%, #000000 100%);
```
Comment (DohyungLim) [inline src/css/profile/wearable/changeable/theme-circle/listview.less]: Should I use the value of a as 0.00?
I think it is good to use it as 0.

**Review by DohyungLim — COMMENTED** [review]
4 comment.

File: examples/wearable/UIComponents/contents/presentation-views/list/main/sub-text/1-line-text-max-fadeout-left.html (hunk 1 above)
Comment (grzegorz-czajkowski) [reply examples/wearable/UIComponents/contents/presentation-views/list/main/sub-text/1-line-text-max-fadeout-left.html:30]: This text is copied from guide.

File: src/css/profile/wearable/changeable/theme-circle/listview.less (hunk 2 above)
Comment (grzegorz-czajkowski) [reply src/css/profile/wearable/changeable/theme-circle/listview.less]: Fixed.

File: src/css/profile/wearable/changeable/theme-circle/listview.less (hunk 3 above)
Comment (grzegorz-czajkowski) [reply src/css/profile/wearable/changeable/theme-circle/listview.less]: Fixed.

File: src/css/profile/wearable/changeable/theme-circle/listview.less (hunk 4 above)
Comment (grzegorz-czajkowski) [reply src/css/profile/wearable/changeable/theme-circle/listview.less]: I copied all values from guide but I think adding double notation is not necessary. Thanks.

**Review by lslachciak — COMMENTED** [review]
I have some questions regarding fadeout left example
image
- there is some rendering issue on the left (white dot visible also in Chrome browser), can we eliminate it?
- left fadeout example has right fadeout too. was this intended?

**Comment by grzegorz-czajkowski** [comment]
@lslachciak 
>   there is some rendering issue on the left (white dot visible also in Chrome browser), can we eliminate it?

added correct left position (previously I took it from wrong parent). Similar issue can be seen for fadeout-right, for short time white dot appears on right side (before marquee animation starts). This is related with additional gradient applied by marquee element.

> left fadeout example has right fadeout too. was this intended?

The right fadeout comes from marquee animation which is applied by default.

**Review by lslachciak — APPROVED** [review]

**Pull request merged**